media query and animal content short summary

PriyeshIllam opened the pull request:
Fixed the media query for Video welcome message
Added the short description animal page

PriyeshIllam committed:
media query and animal content short summary

Ankitasingh88 commented on scripts/index.js:
```diff
@@ -21,62 +21,72 @@ groups.forEach(group => {
 
 
 let mammalquokka  = new Animals("Mammals", "Quokka", 10, ["Plant eaters", " shrubs", "grasses"],
     "he Quokka, Setonix brachyurus, was described by early Dutch explorer, Willem de Vlamingh, 'as a kind of rat as big as a common cat'. His first sighting of the Quokka was on an island off the mouth of the Swan River. He named the island Rottenest ('rat nest') in honour of this sighting. The island is now known as Rottnest Island. Essentially the Quokka looks very much like other wallabies.It  ...
```
Letter T is missing at beginning of the sentence

PriyeshIllam replied:
Fixed. Thanks

Ankitasingh88 commented on styles/index.css:
```diff
@@ -164,4 +183,132 @@
+@media (max-width: 480px) {
+  /* Sidebar occupies the first row */
+  .sidebar {
+    width: 100%; /* Full width for the sidebar */
```
The width of the sidebar is different for each group. An alternate may be to reduce font size and have the width fixed.

PriyeshIllam replied:
when the screen size is lesss than 768 the sidebar is moved to the top rather than on the left. That is why this widht is changed

Ankitasingh88 commented on styles/index.css:
```diff
@@ -164,4 +183,132 @@
+  flex-direction: column;
+}
+
+ .main-content{
```
When we click on a animal in mobile view, you dont actually see anything unless you scroll down. May be one option would be to roll up the sidebar somehow

PriyeshIllam replied:
Fixed. Thanks

VaishaliGovindaraj commented on scripts/index.js:
```diff
@@ -21,62 +21,72 @@ groups.forEach(group => {
     }
-    document.querySelector('.welcome-container').style.display = 'flex'; // show the welcome message on click
+
+    if (window.innerWidth <= 768) {
```
when an animal is clicked(selected) and section dropdown is closed --> welcome content is displayed(this is as expected) but the selected animal not getting deselected

![dropdown_deselect](https://github.com/user-attachments/assets/4ca847b2-e8f4-4cc0-89fe-781d72fd85f2)

PriyeshIllam committed:
scroll to the animal summary on mobile screen

PriyeshIllam committed:
Merge branch 'develop' into sidebar_menu_priyesh

Lvhacker1 commented on styles/layout.css:
```diff
@@ -241,14 +241,158 @@ body {
         height: auto;
+    }*/
+
+    /*footer panels and copyright*/
```
???

Ankitasingh88 reviewed (approved):
thanks for fixing the comments

Lvhacker1 reviewed (changes requested):
row 254-292, what is the styling for?

Lvhacker1 commented on index.html:
```diff
@@ -103,13 +103,12 @@ <h1>Welcome to DJURPARK</h1>
 
                 <div class="animal-container" id="animal-container">
+                    <img src="./images/Echidna.jpg" class="animal-image"> 
                     <h1 class="animal-name"></h1>
```
Can you change this to h2 instead of h1, since h1 is going to be used for the page title.

PriyeshIllam committed:
h2 updatedh2 updated

PriyeshIllam committed:
h2 updated:wq
Merge branch 'sidebar_menu_priyesh' of github.com:VaishaliGovindaraj/JS_Zoo_GroupProject into sidebar_menu_priyesh

VaishaliGovindaraj reviewed (approved):
looks good to me

PriyeshIllam committed:
deleted footer styling
